.github/workflows/init_repository.yml

```diff
@@ -0,0 +1,110 @@
+          echo "SNAPSHOT_KEY=gcpkms://projects/sigstore-root-signing/locations/global/keyRings/root/cryptoKeys/snapshot" >> $GITHUB_ENV
+          echo "TIMESTAMP_KEY=gcpkms://projects/sigstore-root-signing/locations/global/keyRings/root/cryptoKeys/timestamp" >> $GITHUB_ENV
+          echo "PREV_REPO=${{ inputs.prev_repo }}" >> $GITHUB_ENV
+          # Note: we set LOCAL=1 because we manually push the changes in the next job.
+          echo "LOCAL=1" >> $GITHUB_ENV
+      - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v2.2.0
+        with:
+          go-version: 1.18.x
+      - uses: google-github-actions/setup-gcloud@877d4953d2c70a0ba7ef3290ae968eb24af233bb # v0.5.1
+        with:
+          project_id: sigstore-root-signing
+      # Setup OIDC->SA auth
+      - uses: google-github-actions/auth@dac4e13deb3640f22e3ffe758fd3f95e6e89f712 # v0.7.2
+        id: auth
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/163070369698/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider'
+          service_account: 'github-actions@sigstore-root-signing.iam.gserviceaccount.com'
+          create_credentials_file: true
+      - name: Login
+        run: |
+          gcloud auth login --brief --cred-file="${{ steps.auth.outputs.credentials_file_path }}"
+          gcloud auth list
+
+      # Initialize the new root and targets layout
+      - name: step 0
+        # Note that we are already in a git repository and are manually pushing changes with the job below
+        # so we can simply build the TUF binary and bypass git setup in step 0.
+        run: |
+          go build -o tuf -tags=pivkey ./cmd/tuf
+      - name: step 1.5
+        run: |
+          ./scripts/step-1.5.sh ${{ inputs.revoke_key }}
+      - name: Upload new repository
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v2.3.1
+        with:
+          name: root-and-targets
+          path: repository/repository/
+          retention-days: 5
+
+  push:
+    needs: init_root
+    runs-on: ubuntu-20.04
+    permissions:
+      pull-requests: 'write'
+      contents: 'write'
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v2.4.0
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741 # v2.1.0
+        with:
+          name: root-and-targets
+          path: repository/repository/
+      # Open pull request changes
+      - name: create pull request
+        uses: peter-evans/create-pull-request@171dd555b9ab6b18fa02519fdfacbb8bf671e1b4 # v4.1.2
+        with:
+          commit-message: update root and targets
+          title: Update Root and Targets
+          body: Initializes a new root and targets
+          branch: init-root-targets
+          signoff: true
+          reviewers: asraa,dlorenc
```

Comment:
can you add me?

Reply:
and any other folks who are active with write access? same with snapshot-timestamp.yml

Reply:
sure: not totally concerned since it opens a PR and anyone can/should review